test/test_backbone_utils.py

```diff
@@ -16,7 +17,7 @@ def get_available_models():
     return [k for k, v in models.__dict__.items() if callable(v) and k[0].lower() == k[0] and k[0] != "_"]
 
 
-@pytest.mark.parametrize("backbone_name", ("resnet18", "resnet50"))
+@pytest.mark.parametrize("backbone_name", resnet.__all__[1:])
```

Comment:
Adding all possible variants of ResNet is increases the execution time of the text by 12x. All of the variants are constructed using the same class, which minimizes the chance of one of them being incompatible. Please also note that the `resnet_fpn_backbone` method aim to support a handful of ResNet variants which are used in our detection models (not ResNeXt, WideResNets and other architectures that you test now). 

I recommend reverting this change as what we had previously is a good tradeoff between cost/speed and coverage.

Reply:
I thoughts that tests should be check everting, and not maintain a good ratio of speed and coverage.  
Also when someone will add model to the resnet family, it will check it.

Reply:
I understand. Unfortunately making this test 12x slower is not an option due to the cost. The backbones that are currently passed to this test are those that use the utility so it's a reasonable tradeoff given the constraints. If you revert the two changes I highlights, I think we are in position to merge the PR. Thanks!

Reply:
We should add at least resnet101.

Reply:
@datumbox
reasons to add resnet101:
- resnet18, resnet 50, and resnet101, is the most common backbond for detection.
- It have better ration of cost/speed and coverage.
- we check 3 classes on mobilenet, and for consistency we check here also 3 classes. 
You cannot beat it with your logic.

Reply:
@datumbox Please comment.

Reply:
@talregev As explained earlier, increasing the execution times on CI without a very good reason is not an option. Our team is working hard to reduce the duration of our tests as we must reduce the costs (see #5574 for related efforts). This PR is expected to increase the execution times as it contains extra model initializations. Therefor this PR can't be merged until these extra model initializations are decreased.

Reply:
@datumbox I give you 3 reason why, you just Ignore them.
Also, improve test also increasing the execution times. It the fact that you test something, it cost time. 
I remove the resenet101. But I think it should added.